packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -219,12 +219,13 @@ class VideoEvent {
   ///
   /// The [eventType] argument is required.
   ///
-  /// Depending on the [eventType], the [duration], [size] and [buffered]
-  /// arguments can be null.
+  /// Depending on the [eventType], the [duration], [size], [rotation], and
+  /// [buffered] arguments can be null.
```

Comment:
These changes will need to be split out per https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-federated-plugins

Reply:
Ack. I'll fetch latest changes and split the PR when the other content is approved. Thanks!